create/how_to/search_bar_for_docs.md

```diff
@@ -179,7 +179,7 @@ These three fields are mandatory, but more [optional fields are available](https
 ::: warning
 
 Since the configuration file is public, we strongly recommend providing the MeiliSearch public key in a production environment, which is enough to perform search requests.
```

Comment:
```suggestion
Since the configuration file is public, we strongly recommend providing the built-in Default Search API key in a production environment, which is enough to perform search requests.
```